README.md

```diff
@@ -50,6 +53,13 @@ cssStyleObserver.attach(document.body);  /* Attach observer to `document.body` *
 cssStyleObserver.detach();               /* Detach observer */
 ```
 
+### Callback Modes
+
+The CSSStyleObserver supports different modes for invoking the callback, specified via the callbackMode option in the options object:
+
+* CallbackMode.INDIVIDUAL: Only the changed property is passed to the callback function
+* CallbackMode.ALL: All observed properties are passed to the callback function, regardless of whether they changed or not
+
 Try out a demo on CodePen: [https://codepen.io/bramus/pen/WNqKqxj](https://codepen.io/bramus/pen/WNqKqxj?editors=1111)
```

Comment:
This line is part of the previous section, not the “Callback Modes” section.

Reply:
updated